src/lib/plugins/column-filter/column-filter.component.ts

```diff
@@ -48,10 +52,22 @@ export class ColumnFilterComponent implements OnInit {
 
 	ngOnInit() {
 		const { model } = this.plugin;
-		const key = this.column.key;
+		const { column } = this;
+		const { key } = column;
 		const context = { key };
 
 		const columnFilter = new ColumnFilterView(model, context);
+		this.filterOperators = model.filter().operators(column);
```

Comment:
can we make getter? get operators() { return model.filter().operators(this.column); }